### PR TITLE
move scripts to bin

### PR DESCRIPTION
**Pull request by garyverhaegen-da**
Because bin is part of the PATH (thanks to .envrc), this makes those commands work from anywher in the project, just like the `deps` command.

Also updated the README to match and added a `clean` command that removes the `workdir` folder.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -60,77 +61,66 @@ export ARTIFACTORY_USERNAME=firstname.lastname
...
-
-:warning: If you do make a change to the TOCs in this repo, you then have to duplicate the change in the relevant satellite repo.
+Run the `live-preview` script to render a local view of the site.
 
 ```zsh
-./live-preview.sh
+live-preview
 ```
 
+The html files are in `workdir/build/gen`. Rerun the script to view any edits to `docs/` files or the top-level ToC (`_toc.yml`).
```
Comment (garyverhaegen-da) [inline README.md:85]: Rerunning the script should ideally not be necessary here but I'll leave that here for the time being. Please test and update accordingly in a follow-up PR if needed.

File: README.md [hunk 2]
```diff
@@ -158,12 +148,13 @@ For example:
...
 ```
 
-4. Update the `LATEST` file to include the snapshot version containing the changed PR. 
+4. Update the `LATEST` file to include the snapshot version containing the changed PR.
 
 ```json
 {
   "daml": "2.1.0-snapshot.20220311.9514.0.ab6e085f",
   "canton": "20220315",
+  "daml_finance"; "1.0.4",
```
Comment (katmurp) [inline README.md:157]:
```suggestion
  "daml_finance": "1.0.4",
```

File: README.md [hunk 3]
```diff
@@ -60,77 +61,66 @@ export ARTIFACTORY_USERNAME=firstname.lastname
...
-cd docs
-```
-
 ### Download docs
 
 Run the following script to download the documentation tarballs from the Daml and Canton repos:
 
 ```zsh
-./download.sh
+download
```
Comment (katmurp) [inline README.md:74]: wasn't able to run these but assuming the change is in this PR...so will test later ...

**Review by katmurp — CHANGES_REQUESTED** [review]

File: README.md (hunk 3 above)
Comment (garyverhaegen-da) [reply README.md:74]: Yes, the change is in this PR.

File: README.md (hunk 2 above)
Comment (garyverhaegen-da) [reply README.md:157]: Good catch, thanks!

**Review by katmurp — APPROVED** [review]

**Pull request merged**